Pipfile

```diff
@@ -30,6 +30,7 @@ pytest = "*"
 pre-commit = "*"
 black = "*"
 reorder-python-imports = "*"
+mitmproxy = "*"
```

Comment:
no mypy?

Reply:
huh?? i guess my changes got discarded, fixing

Reply:
fixed, looking like a squash and merge angle